website/_data/compatibility.yml

```diff
@@ -16,6 +25,7 @@ releases:
       - "2.38.0"
   - version: 0.23.0
     compatible:
+      - "2.41.0"
```

Comment:
This one might have been missed due to manual editing of the file. This PR was generated using the `generate-version-compatibility-overview.sh` changes in #1927.